src/views/Upload.vue

```diff
@@ -75,7 +75,26 @@ export default {
 	},
 	methods: {
 		updateFileList(fileArray) {
-			this.files = [...this.files, ...fileArray];
+			for (var i = 0; i < fileArray.length; i++) {
+				if (fileArray[i].size > 20*1024*1024){
+					this.$store.dispatch('AlertStore/setAlert', {
+						message:
+							'Súbory väčšie ako 20MB nie sú podporované!',
+						type: 'error',
+					});
+					
+			} else {
+				if (this.files.length <= 50) {
```

Comment:
So, according to this logic, if current length of files is 48, and you add 4 more files, the error would be displayed twice. I don't think its correct and the error should be shown only once.